pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -90,6 +103,94 @@ func (vm *Windows) initializeBootstrapperFiles() error {
 	return nil
 }
 
+// handleHybridOverlay ensures that the hybrid overlay is running on the node
+func (vm *Windows) handleHybridOverlay(nodeName string) error {
+	// Check if the hybrid-overlay is running
+	_, stderr, err := vm.Run("Get-Process -Name \""+hybridOverlayProcess+"\"", true)
+
+	// stderr being empty implies that hybrid-overlay was running.
+	if err == nil || stderr == "" {
+		// Stop the hybrid-overlay
+		stopCmd := "Stop-Process -Name \"" + hybridOverlayProcess + "\""
+		_, stderr, err := vm.Run(stopCmd, true)
+		if err != nil || stderr != "" {
+			log.Info("%s:\n%s", stopCmd, stderr)
```

Comment:
This can be reformatted using key value pair format.

Reply:
:+1: on using the key value pairs instead of manually adding newlines